source/PhpUnitsOfMeasure/PhysicalQuantity/Mass.php

```diff
@@ -103,6 +101,19 @@ function ($x) {
         $new_unit->addAlias('pounds');
         $this->registerUnitOfMeasure($new_unit);
 
+        // Ton (imperial)
+        $new_unit = new UnitOfMeasure(
+            'ton',
+            function ($x) {
+                return $x / 4.535924e2 * 2;
+            },
+            function ($x) {
+                return $x * 4.535924e2 * 2;
```

Comment:
Is the math here right? Order of operations would divide, then multiply by two, whereas the second one would multiply and then multiply by two.

For that matter, wouldn't it be much clearer to just use the actual number `907.185`?